packages/govuk-frontend/src/govuk/components/character-count/character-count.test.js

```diff
@@ -1,20 +1,14 @@
 const { goToComponent, renderAndInitialise } = require('govuk-frontend-helpers/puppeteer')
 const { getExamples } = require('govuk-frontend-lib/files')
 
-const jestPuppeteerConfig = require('../../../../../../jest-puppeteer.config')
-
-// Detect when browser has been launched headless
-const { headless = 'new' } = jestPuppeteerConfig.launch
-
 // The longest possible time from a keyboard user ending input and the screen
 // reader counter being updated: handleFocus interval time + last input wait time
-// but raised to the nearest whole second when browser is not headless as timers
-// in background pages will be aligned, clamped or throttled
-const debouncedWaitTime = headless ? 1500 : 2000
+// but raised by 50ms to avoid the timeout window
+const debouncedWaitTime = 1550
```

Comment:
Since https://github.com/alphagov/govuk-frontend/pull/3543 (or perhaps in recent Puppeteer updates) we're seeing much more accurate timers

Running assertions at the exact `setTimeout()` period can fail the test so I've added 50ms on